packages/components/src/navigation/item/index.js

```diff
@@ -70,34 +64,23 @@ export default function NavigationItem( props ) {
 		onClick( event );
 	};
 	const icon = isRTL ? chevronLeft : chevronRight;
+	const baseProps = isText
+		? { as: 'div', ...restProps }
```

Comment:
This `as: 'div'` is unnecessary, given `ItemBaseUI` renders as `div` by default, but I think there's also value in being clear.
No strong opinions about this: feel free to keep it or remove it as you like. 🙂